index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang ="en">
+    <head>
+        <title>Daniel Hopton</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/df6431e323547add1b4cf45992913f15286456d3/devicon.min.css">
+    </head>
+    <body>
+        <nav>
+            <div id="navwrapper">
+                <div class="navitem">
+                    <h4>Section Header</h4>
+                    <p>Section Description</p>
+                </div>
+                <div class="navitem">
+                    <h4>Section Header</h4>
+                    <p>Section Description</p>
+                </div>
+                <div class="navitem">
+                    <h4>Section Header</h4>
+                    <p>Section Description</p>
+                </div>
+            </div>
+        </nav>
+        <header>
+            <div id="headerwrapper">
+                <div id ="headertext">
+                    <h1>FIRSTNAME SURNAME</h1>
+                    <h2>Job Title</h2>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                </div>
+                <div id ="badges">
+                    <i class="devicon-csharp-plain"></i>
+                    <i class="devicon-html5-plain"></i>
+                    <i class="devicon-css3-plain"></i>
+                </div>
+            </div>
+        </header>
+        <div id="showcase">
+            <div id ="showcasewrapper">
+                <div id="showcasetext">
+                    <h2>PROJECT NAME</h2>
+                    <h3>PROJECT ITEM</h3>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                </div>
+                <div id="showcaseviewer">
+                    <button class="showcasenav" id="showcaseleft">&lt;</button>
+                    <button class="showcasenav" id="showcaseright">&gt;</button>
+                    <div id="showcasebottom">
+                        <button id="showcaseview">View Project</button>
```

Comment:
make this an anchor not a button as its a link. then you wont need as much css like `background: transparent;`